lib/pages/recording_editor.dart

```diff
@@ -185,47 +185,121 @@ class _RecordingEditorMap extends StatelessWidget {
   final List<LatLng> pointList;
 
   @override
-  Widget build(BuildContext context) {
-    final bounds = LatLngBounds.fromPoints(pointList);
+  _RecordingEditorMapState createState() => _RecordingEditorMapState();
+}
 
-    /*
+LatLngBounds _getBoundsFromPoints(List<LatLng> points) {
+  final bounds = LatLngBounds.fromPoints(points);
+
+  /*
     When all the points are the same the southWest and northEast
     become the same too. This leads to an error in flutter map.
     (Unsupported operation: Infinity or NaN toInt)
     This normally only happens during debugging.
-     */
+  */
 
-    if (bounds.southWest!.latitude == bounds.northEast!.latitude) {
-      bounds.southWest!.latitude += 0.0005;
-      bounds.northEast!.latitude -= 0.0005;
-    }
+  if (bounds.southWest!.latitude == bounds.northEast!.latitude) {
```

Comment:
I dont really understand this check to be honest. Does this mean that the window has zero width or height ?

Reply:
Well the bounds have 0 distance between them. And I can't tell you exactly but either flutter_map thinks the coordinates wrap around the whole world or it is actually a zero width issue.